### DISCUSSION TITLE
Does SkyWalking Satellite support  browser (BrowserPerf) ?

### DISCUSSION THREAD

**Discussion by unihon** [apache/skywalking#11700] [Q&A]
Does SkyWalking Satellite support  browser (BrowserPerf) ?
I want SkyWalking Satellite  receive browser  metrics, traces, and logs (from skywalking-client-js) and send them to kafka/oap.

**Comment by mrproliu** [comment]
Nope, The satellite does not support receiving browser perf protocol data. 
In addition, does `skywalking-client-js` transmit obs data via HTTP? If so, the satellite only supports transmission using gRPC protocol and does not support HTTP.

**Reply by wu-sheng** [reply]
As browser agent is using HTTP 1.x, any proxy works, pick Nginx, Envoy, etc.

**Comment by unihon** [comment]
![image](https://github.com/apache/skywalking/assets/42078581/6d120796-65c8-4c88-8180-936038499d3f)

I need a transport layer to transmit the agent's http/grpc data to a buffer layer (es). oap actively pulls the data from the buffer layer, and the data is processed and saved to es.

I have tested the methods of SkyWalking Satellite (olny native log support send to kafka) and agent to upload kafka（some agents don't support ）, but neither can realize my application scenario.

**Reply by wu-sheng** [reply]
No, we don't support that.

**Reply by wu-sheng** [reply]
Satellite is for load balance, rather than to Kafka. If you need to transfer for Kafka, that is agent or your private work, so far.
